bin/pygrb/pycbc_pygrb_plot_coh_ifosnr

```diff
@@ -175,9 +169,8 @@ init_logging(opts.verbose, format="%(asctime)s: %(levelname)s: %(message)s")
 
 # Check options
 trig_file = os.path.abspath(opts.trig_file)
-found_file = (
-    os.path.abspath(opts.found_missed_file) if opts.found_missed_file else None
-)
+found_file = os.path.abspath(opts.found_missed_file)\
+    if opts.found_missed_file else None
```

Comment:
Please do not change this formatting. In fact, I encourage everyone to run `black --skip-string-normalization --line-length 79` to apply a consistent code style.